typerep-map.cabal

```diff
@@ -101,6 +103,7 @@ benchmark typerep-map-benchmark
   main-is:             Main.hs
   other-modules:       CMap
                      , CacheMap
+                     , Common
```

Comment:
```suggestion
                     , Spec
```

Reply:
I guess CI fails because of this

Reply:
Thanks!

Reply:
Still fails because of `Hack` leftovers somewhere..